cmd/server-mux_test.go

```diff
@@ -109,20 +109,18 @@ func dial(addr string) error {
 
 // Tests initializing listeners.
 func TestInitListeners(t *testing.T) {
-	portTest1 := getFreePort()
-	portTest2 := getFreePort()
```

Comment:
@harshavardhana I am aware of how getFreePort works. It is obvious from the travis-ci failure that the port returned by getFreePort wasn't really free. getFreePort closes the listener (and the underlying socket) leaving the port free for other applications to bind to. So, getFreePort doesn't guarantee that the port it finds free to be available indefinitely. This is why the unit test failed. Whether port will remain free during the test is non-deterministic and can't be relied on. Does this satisfactorily explain the problem?